java/src/main/bin/build.sh

```diff
@@ -8,10 +8,10 @@ __brew_not_installed=2
 
 make_base="cd /vowpal_wabbit;
 make clean;
-make;"
+make vw java;"
 
 ubuntu_base="apt-get update -qq;
-apt-get install -qq software-properties-common g++ make libboost-all-dev default-jdk;"
+apt-get install -qq software-properties-common g++ make libboost-program-options-dev default-jdk;"
```

Comment:
I forgot to mention that once we just use `libboost-program-options-dev` we also need to include `zlib1g-dev`.  The whole line should read

```
apt-get install -qq software-properties-common g++ make libboost-program-options-dev zlib1g-dev default-jdk;"
```